systemd/ua-status.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Ubuntu Advantage status refresh
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/ua status
```

Comment:
Should we follow the apt-daily.{service,timer} units?

Reply:
`We probably want to sync correct service status information before any non-root user logs into the system to avoid exposure for invalid satus information (because only root user *ua status* runs would attempt to  actually update status cache). As such, we probably don't want to delay this operation by a variable amount of time because we might get spurious bugs filed if that timer splay was large enough on one specific reboot.